tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -449,106 +449,36 @@ batch:
-          AES_GCM_SELECTCHECK_START: 1
-          AES_GCM_SELECTCHECK_END: 140
-
-    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_141_231
-      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-20.04_clang-10x_formal-verification.yml
-      env:
-        type: LINUX_CONTAINER
-        privileged-mode: false
-        compute-type: BUILD_GENERAL1_2XLARGE
-        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-10x_formal-verification_latest
-        variables:
-          AES_GCM_SELECTCHECK: 1
-          AES_GCM_SELECTCHECK_START: 141
-          AES_GCM_SELECTCHECK_END: 231
-
-    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_232_289
-      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-20.04_clang-10x_formal-verification.yml
-      env:
-        type: LINUX_CONTAINER
-        privileged-mode: false
-        compute-type: BUILD_GENERAL1_2XLARGE
-        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-10x_formal-verification_latest
-        variables:
-          AES_GCM_SELECTCHECK: 1
-          AES_GCM_SELECTCHECK_START: 232
-          AES_GCM_SELECTCHECK_END: 289
-
-    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_290_325
-      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-20.04_clang-10x_formal-verification.yml
-      env:
-        type: LINUX_CONTAINER
-        privileged-mode: false
-        compute-type: BUILD_GENERAL1_2XLARGE
-        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-10x_formal-verification_latest
-        variables:
-          AES_GCM_SELECTCHECK: 1
-          AES_GCM_SELECTCHECK_START: 290
-          AES_GCM_SELECTCHECK_END: 325
-
-    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_326_356
-      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-20.04_clang-10x_formal-verification.yml
-      env:
-        type: LINUX_CONTAINER
-        privileged-mode: false
-        compute-type: BUILD_GENERAL1_2XLARGE
-        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-10x_formal-verification_latest
-        variables:
-          AES_GCM_SELECTCHECK: 1
-          AES_GCM_SELECTCHECK_START: 326
-          AES_GCM_SELECTCHECK_END: 356
+        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-10x_formal-verification-saw-aarch64_latest
 
-    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_357_384
-      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-20.04_clang-10x_formal-verification.yml
+    # NSym proofs on platform AArch64
+    - identifier: ubuntu2204_clang14x_formal_verification_nsym_aarch64
+      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-22.04_clang-14x_formal-verification-nsym-aarch64.yml
```

Comment:
Nit: We could use [`run_simple_target.yml`](https://github.com/aws/aws-lc/blob/e8d7ca8be56c8972a09aaed990920b2a2b71202b/tests/ci/codebuild/common/run_simple_target.yml) to directly call the test script, and do away with the additional yml files. 
https://github.com/aws/aws-lc/blob/e8d7ca8be56c8972a09aaed990920b2a2b71202b/tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml#L28-L36